tests/sdk/yaml/await-daemonset/Pulumi.yaml

```diff
@@ -0,0 +1,35 @@
+name: await-daemonset
+runtime: yaml
+plugins:
+  providers:
+    - name: kubernetes
+      path: ../../../../bin
```

Comment:
I thought the provider linking happened automatically but I frankly don't know how.  And why don't we see it in the other steps?

Reply:
Yeah it happens automatically in tests, although sometimes I will manually run examples in which case this helps. I took it out since I don't need to test it manually anymore.